transfocate/tests/__init__.py

```diff
@@ -3,6 +3,6 @@
 from . import test_transfocate
 
 # Create a Transfocator with a few inserted lenses
-transfocator = test_transfocate.transfocator()
+transfocator = test_transfocate.make_fake_transfocator()
```

Comment:
I was trying to figure out where this was being used as it doesn't appear to be in the test suite:

Looks like it's here:
```
docs/source/index.rst:    import transfocate.tests
docs/source/index.rst:    mfx_transfocator = transfocate.tests.transfocator
```

Reply:
Ahh, I never bothered trying to track down the usage. I just saw the exception and decided to resolve it without further investigation.